src/workflows/airqo_etl_utils/satellite_data_utils.py

```diff
@@ -0,0 +1,229 @@
+from datetime import datetime, timedelta
+import datetime as dt
+from google.cloud import bigquery
+from google.oauth2 import service_account
+import pandas as pd
+import numpy as np
+import os
+import random
+import string
+import ee
+from functools import reduce
+from pymongo import MongoClient
+from bson.json_util import dumps
+
+from pymongo import MongoClient
+
+from .config import Config
+
+credentials = ee.ServiceAccountCredentials(
+    key_file=Config.GOOGLE_APPLICATION_CREDENTIALS,
+    email=Config.GOOGLE_APPLICATION_CREDENTIALS_EMAIL,
+)
+class DataHandler:
+    def __init__(self):
+        self.client = bigquery.Client()
+        self.data_path = None
+        ee.Initialize(credentials, project=Config.GOOGLE_CLOUD_PROJECT_ID)
+        self.mongo_client = MongoClient(Config.MONGODB_URI)
+        self.db = self.mongo_client[Config.MONGODB_DATABASE]
```

Comment:
```suggestion
```
@wabinyai there is already a connection created at the bottom of `config.py` so you can imiport it and skip this. perhaps create a new collection only.